tachyon/math/finite_fields/binary_fields/binary_fields.h

```diff
@@ -5,9 +5,60 @@
 #ifndef TACHYON_MATH_FINITE_FIELDS_BINARY_FIELDS_BINARY_FIELDS_H_
 #define TACHYON_MATH_FINITE_FIELDS_BINARY_FIELDS_BINARY_FIELDS_H_
 
+#include "tachyon/math/finite_fields/binary_fields/binary_field16_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field1_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field2_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field32_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field4_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field64_config.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field8_config.h"
+#include "tachyon/math/finite_fields/binary_prime_field_traits_forward.h"
+
+namespace tachyon::math {
+
+template <>
+struct BinaryPrimeFieldTraits<BinaryField1Config> {
+  using SubConfig = BinaryField1Config;
+};
+
+template <>
+struct BinaryPrimeFieldTraits<BinaryField2Config> {
+  using SubConfig = BinaryField2Config;
+};
+
+template <>
+struct BinaryPrimeFieldTraits<BinaryField4Config> {
+  using SubConfig = BinaryField4Config;
+};
+
+template <>
+struct BinaryPrimeFieldTraits<BinaryField8Config> {
+  using SubConfig = BinaryField8Config;
+};
+
+template <>
+struct BinaryPrimeFieldTraits<BinaryField16Config> {
+  using SubConfig = BinaryField8Config;
+};
+
+template <>
+struct BinaryPrimeFieldTraits<BinaryField32Config> {
+  using SubConfig = BinaryField16Config;
+};
+
+template <>
+struct BinaryPrimeFieldTraits<BinaryField64Config> {
+  using SubConfig = BinaryField32Config;
+};
+
+}  // namespace tachyon::math
+
 #include "tachyon/math/finite_fields/binary_fields/binary_field1.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field16.h"
 #include "tachyon/math/finite_fields/binary_fields/binary_field2.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field32.h"
 #include "tachyon/math/finite_fields/binary_fields/binary_field4.h"
+#include "tachyon/math/finite_fields/binary_fields/binary_field64.h"
```

Comment:
How about organizing the headers in order of 1, 2, 4, 8, 16, 32, 64 using clang-format off?